### ISSUE TITLE
Adding separate query and test sequences

### ISSUE THREAD

**Issue by nikhita-damaraju** [marbl/ModDotPlot#6]
Hi,
I was wondering if there was a way to make the static plot for two separate sequences? As I understand it right now, it checks the identity of a sequence with itself.

**Comment by tchrisboles** [comment]
Same question as Nikhita above. Thanks.

**Comment by tchrisboles** [comment]
As in comparing pseudogenes or segdups on different chromosomes?

**Comment by nikhita-damaraju** [comment]
@tchrisboles , yes that could be one of the outcomes. I was thinking more along the lines of showing the alignment of query sequence to a reference sequence.

**Comment by alexsweeten** [comment]
Hi @nikhita-damaraju and @tchrisboles,

Sorry for the late response! I've added this functionality in the most recent update to ModDotPlot. To create an a vs. b style dotplot, add the `--compare `  command line argument when creating static plots or using interactive mode.

**Comment by tchrisboles** [comment]
Sorry for the delay. Just tried A vs B dotplot with --compare-only option. Works great. Thanks.
Chris

**Comment by tchrisboles** [comment]
Actually, one other thing. What's the proper way to exit interactive mode?

**Comment by alexsweeten** [comment]
Running ctrl+C on the terminal ModDotPlot is being run on will exit interactive mode and kill the process. 

Adding a proper exit button might be cleaner. Will add this in the next version. Thanks and glad it's working well!

- Alex
